cloudsql-mysql-plugin/docs/CloudsqlMysql-batchsink.md

```diff
@@ -0,0 +1,155 @@
+    | FLOAT                          | float                 |                                                    |
+    | DOUBLE                         | double                |                                                    |
+    | DECIMAL                        | decimal               |                                                    |
+    | DATE                           | date                  |                                                    |
+    | DATETIME                       | timestamp             |                                                    |
+    | TIMESTAMP                      | timestamp             |                                                    |
+    | TIME                           | time                  |                                                    |
+    | YEAR                           | date                  |                                                    |
+    | CHAR                           | string                |                                                    |
+    | VARCHAR                        | string                |                                                    |
+    | BINARY                         | bytes                 |                                                    |
+    | VARBINARY                      | bytes                 |                                                    |
+    | TINYBLOB                       | bytes                 |                                                    |
+    | TINYTEXT                       | string                |                                                    |
+    | BLOB                           | bytes                 |                                                    |
+    | TEXT                           | string                |                                                    |
+    | MEDIUMBLOB                     | bytes                 |                                                    |
+    | MEDIUMTEXT                     | string                |                                                    |
+    | LONGBLOB                       | bytes                 |                                                    |
+    | LONGTEXT                       | string                |                                                    |
+    | ENUM                           | string                |                                                    |
+    | SET                            | string                |                                                    |
+
+
+
+Examples
+------
+**Connecting to a public CloudSQL MySQL instance**
+
+Suppose you want to write output records to "users" table of CloudSQL MySQL database named "prod", as "root" user with 
+"root" password (Get the latest version of the CloudSQL socket factory jar with driver and dependencies 
+[here](https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/releases) and add the driver for MySQL), 
```

Comment:
we don't need to "add the driver for MySQL" right?

Reply:
Correct. I'll remove that part